django/contrib/gis/db/backends/postgis/operations.py

```diff
@@ -260,13 +260,17 @@ def get_distance(self, f, dist_val, lookup_type):
             # Assuming the distance is in the units of the field.
             dist_param = value
 
-        if (not geography and geodetic and lookup_type != 'dwithin'
-                and option == 'spheroid'):
-            # using distance_spheroid requires the spheroid of the field as
-            # a parameter.
-            return [f._spheroid, dist_param]
-        else:
-            return [dist_param]
+        params = [dist_param]
+        # handle_spheroid *might* be dropped in Django 2.0 as PostGISDistanceOperator
```

Comment:
I'd add the ticket reference in the comment so we remember to close the ticket in case the comment is removed.